VAMobile/.detoxrc.json

```diff
@@ -6,8 +6,8 @@
   "apps": {
     "ios": {
       "type": "ios.app",
-      "binaryPath": "ios/output/Build/Products/Release-iphonesimulator/VAMobile.app",
-      "build": "cd ios && xcodebuild -configuration Release -workspace VAMobile.xcworkspace -scheme VAMobileRelease -derivedDataPath ./output -sdk iphonesimulator BUILD_LIBRARIES_FOR_DISTRIBUTION=YES build && cd .."
+      "binaryPath": "ios/output/Build/Products/Debug-iphonesimulator/VAMobile.app",
```

Comment:
FWIW I noticed the Detox [docs](https://wix.github.io/Detox/docs/introduction/project-setup/#step-2-app-configs) include both a debug and release build for `apps` in the config. This would mean we'd need 4 entries though 😅 (2 for each platform). Not sure what their guidance is on when to use either build, but I would think defaulting to the release build would still be preferred in that case for the reasons Chris mentioned